webpack.config.js

```diff
@@ -160,6 +193,27 @@ function getWebpackConfig( {
 					cacheIdentifier,
 					exclude: /node_modules\//,
 				} ),
+				{
+					test: /\.jsx?$/,
+					include: _.memoize( shouldTranspileDependency ),
```

Comment:
The `memoize` here is probably an insignificant optimization. It will maintain a map with several thousands entries, and webpack does so many checks on each filename so that this extra one is unlikely to make any impact.